tests/modules/nf-core/cdhit/cdhit/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { CDHIT } from '../../../../../modules/nf-core/cdhit/cdhit/main.nf'
```

Comment:
Need the new name here too
```suggestion
include { CDHIT_CDHIT } from '../../../../../modules/nf-core/cdhit/cdhit/main.nf'
```

Reply:
Sorry I probably should have started afresh when I realised the directory structure needed to change!